libs/blocks/merch/merch.js

```diff
@@ -683,6 +683,7 @@ export async function getPriceContext(el, params) {
   const displayPerUnit = params.get('seat');
   const displayRecurrence = params.get('term');
   const displayTax = params.get('tax');
+  const displayAnnual = getConfig().locale?.prefix === '/au' && (params.get('annual') || !getMetadata('invert-au-abm-annual'));
```

Comment:
can we check in tomorrow's tech call whether the FF `invert-au-abm-annual` is needed?
my understanding was that GWP was favorable to inverting the default logic in AU, and only let annual price to be rendered explicitly with `&annual=true`

Reply:
We can ask them tomorrow. I will demo this anyway. But if I invert this for the whole /au and then let them to add `&annual=true` where they need the annual price that will require lots of authoring. As far as I know they need this annual price next to the ABM almost everywhere. Australian legal reasons.

Reply:
we should ask, but in parallel, can I suggest  FF name: `mas-enable-price-annual`  and it would be set only for /au tree.  The reason is that later we will have more FFs and we should follow a pattern for naming them.
We can discuss naming convetion, mine is just a suggestion.

Reply:
we can set `mas-enable-price-annual` before PR merge so that it will keep working as before.
In parallel, we can ask GWP to adopt `&annual=true` notation going forward.

Reply:
As we agreed on yesterday's meeting we will not use any metadata flag. I updated the PR description and added the label "do not merge". Waiting for GWP now.